tests/installation/playwright-cli-install-should-work.spec.ts

```diff
@@ -79,6 +81,35 @@ test('install command should work with HTTPS_PROXY', { annotation: { type: 'issu
   await proxy.stop();
 });
 
+test('install command should work with mirror that uses chunked encoding', async ({ exec, checkInstalledSoftwareOnDisk }) => {
+  await exec('npm i playwright');
+  const server = http.createServer(async (req, res) => {
+    try {
+      const upstream = await fetch('https://cdn.playwright.dev/dbazure/download/playwright' + req.url);
+      const headers = new Headers(upstream.headers);
+      headers.delete('content-length');
+      res.writeHead(upstream.status, Object.fromEntries(headers));
+      await upstream.body.pipeTo(Writable.toWeb(res));
+      return;
+    } catch (e) {
+      console.error(e);
+      res.statusCode = 500;
+      res.end(String(e));
+      return;
+    }
+  });
+  await new Promise<void>(resolve => server.listen(0, resolve));
+  await test.step('playwright install chromium', async () => {
```

Comment:
nit: no need for a step, `exec()` does it automatically.

Reply:
we have this throughout the entire file 🤔